src/styles/App.css

```
@@ -10,48 +10,65 @@
 
   /* Color scheme for application */
   --slate-gray: #627c85ff;
-  --alice-blue: #e1eff6ff;
+  --alice-blue: rgba(190, 195, 211, 0.808);
   --english-vermillion: #d33e43ff;
   --raisin-black: #1c1d21ff;
   --blue-ncs: #2e86abff;
-
+  --calander-color: rgba(110, 127, 199, 0.116);
   --title-color-dark:  #464646;
-  --title-color-light:  #072c5f;
-  --shade3-color:  #0E57C5;
-  --shade4-color:  #488CF2;
-  --shade5-color:  #84B1F7;
-  --shade6-color:  #ACCBF9;
+  --title-color-light:  #ffffff;
+  --shade1-color: #ffffff;
+  --shade2-color:  #ffffff;
+  --shade3-color:  #ffffff;
+  --shade4-color:  #cbd2db;
+  --shade5-color:  #ffffff;
+  --shade6-color:  #ff0000;
+  --shade7-color: #ffffff;
   --text-color-black: #000000;
   --text-color-white: #ffffff;
   --table1-color: #ACCBF9;
   --table2-color: #ffffff;
   --border-color: #000000;
-  --background-color: #aaccfc;
+  --background-color: #ffffff;
   --remove-button: #e64e4e;
   --remove-hover-button: #e47c7c;
-  --remove-hove: 
+  --remove-hove: ;
+  /* Discreete Palette */
+  --c1-color: #B4B4CF;
```

These names don't really tell _what_ color it is. 
```suggestion
  --blue-magenta: #B4B4CF;
```

---

I don't recommend fixing this now however. It is to late in the project,  your time is better spent elsewhere

---

Väldigt sant